content/docs/intro/console/accounts-and-organizations/organizations.md

```diff
@@ -17,7 +17,7 @@ with other developers.
 If you're an admin of a Pulumi organization, you have the
 ability to:
 
-* [Add users]({{< relref "../collaboration/organization-roles/#organization-membership" >}})
+* [Invite users]({{< relref "../collaboration/organization-roles/#organization-membership" >}})
```

Comment:
Could we make these URLs all docroot-relative? 🙏 The reason is sort of hard to explain, but the gist is that Hugo occasionally gets confused and fails with really unhelpful error messages when paths aren't strictly unique (e.g., two totally unrelated `relrefs` located in different parts of the content tree, both referencing a subpath of the same name). I'm usually the one who has to hunt these down and fix them when someone hits them, and even I sometimes struggle to figure out what the deal is. 😂 So if we could just `relref "/always/site/root/relative"`, that'll always work.  

```suggestion
* [Invite users]({{< relref "/docs/intro/console/collaboration/organization-roles/#organization-membership" >}})
```